packages/fxa-shared/metrics/glean/fxa-backend-metrics.yaml

```diff
@@ -0,0 +1,204 @@
+---
```

Comment:
@akkomar or @badboy can I get a 👍 from either of you on this?  The metrics here are the same as the frontend ones.

Reply:
Well this is not 100% true now that https://github.com/mozilla/fxa/pull/15538 has been merged.  I'll update backend yamls later today.